src/generate-query.ts

```diff
@@ -467,6 +524,8 @@ function getArgsAndVars(
   const requiredArguments = allArgs.filter((arg) =>
     considerArgument(arg, config)
   )
+  const missingSlicingArg = getMissingSlicingArg(requiredArguments, field)
```

Comment:
Can we add a comment here that clarifies that this feature utilizes the API Connect directives?

Reply:
Will add a comment.